src/devices_models/device_constructors/load_constructor.jl

```diff
@@ -345,7 +343,7 @@ function construct_device!(
     add_to_expression!(
         container,
         ReactivePowerBalance,
-        ReactivePowerTimeSeriesParameter,
+        ActivePowerTimeSeriesParameter,
```

Comment:
Add a note here please about why we do this. 

```suggestion
        # The base assumption is that reactive power follows the same curve as active power
        ActivePowerTimeSeriesParameter,
```